src/pages/memberhsip/components/createYouthProfileForm/CreateYouthProfileForm.tsx

```diff
@@ -292,41 +379,65 @@ function RegistrationForm(props: Props) {
                 as={TextInput}
                 id="guardianEmail"
                 name="guardianEmail"
+                invalid={props.submitCount && props.errors.guardianEmail}
+                invalidText={
+                  props.submitCount &&
+                  props.errors.guardianEmail &&
+                  t(props.errors.guardianEmail)
+                }
                 labelText={t('registration.email')}
               />
               <Field
                 className={styles.formInput}
                 as={TextInput}
-                id="guardianPhoneNumber"
-                name="guardianPhoneNumber"
+                id="guardianPhone"
+                name="guardianPhone"
+                invalid={props.submitCount && props.errors.guardianPhone}
+                invalidText={
+                  props.submitCount &&
+                  props.errors.guardianPhone &&
+                  t(props.errors.guardianPhone)
+                }
                 labelText={t('registration.phoneNumber')}
               />
             </div>
             <h2>{t('registration.confirmSend')}</h2>
             <p>{t('registration.processInfoText')}</p>
             <ul className={styles.acceptTermsCheckBox}>
-              <Field
-                name="acceptTerms"
-                type="checkbox"
-                value="acceptanceTerms"
-              />
+              <Field name="terms" type="checkbox" />
               <span className={styles.listLabel}>
                 {t('registration.acceptTermsText_1')}
                 <a href="/#">{t('registration.acceptTermsText_link')}</a>
                 {t('registration.acceptTermsText_2')}
               </span>
             </ul>
             <button
-              disabled={props.values.acceptTerms.length === 0}
+              disabled={
+                !props.values.terms ||
+                !validateDate(
+                  props.values.birthYear,
+                  props.values.birthMonth,
+                  props.values.birthDay
+                )
+              }
               type="submit"
             >
               {t('registration.sendButton')}
             </button>
+            <div className={styles.birhtdayErrorMessage}>
```

Comment:
The error is now shown before the user has even touched the form, should probably be hidden in the beginning.